modular_skyrat/modules/customization/_globalvars/lists.dm

```diff
@@ -83,6 +83,12 @@ GLOBAL_LIST_INIT(preference_breast_sizes, list(
 	BREAST_SIZE_H,
 	BREAST_SIZE_I,
 	BREAST_SIZE_J,
+	BREAST_SIZE_K,
+	BREAST_SIZE_L,
+	BREAST_SIZE_M,
+	BREAST_SIZE_N,
+	BREAST_SIZE_O,
+	BREAST_SIZE_P,
```

Comment:
If you're just gonna add everything to this list, just remove it, and compare to `GLOB.breast_size_to_number.keys` instead. Less code to maintain, and frees up teeny bit of resources.